dll/win32/browseui/explorerband.cpp

```diff
@@ -147,7 +148,7 @@ HRESULT _ReparsePIDL(LPITEMIDLIST buggyPidl, LPITEMIDLIST *cleanPidl)
 CExplorerBand::CExplorerBand()
     : m_pSite(NULL)
     , m_fVisible(FALSE)
-    , m_bNavigating(FALSE)
+    , m_bNavigating(0)
```

Comment:
Since this is no longer a Boolean, please rename this something else. Maybe m_mtxNavigating (Navigating Mutex) or something like that.